modules/publicize/assets/gutenberg_client/publicize-form.jsx

```diff
@@ -0,0 +1,192 @@
+/**
+ * Publicize sharing form component.
+ *
+ * Displays text area and connection list to allow user
+ * to select connections to share to and write a custom
+ * sharing message. Dispatches publicize form data to
+ * editor post data in format to match 'publicize' field
+ * schema defined in{@see class-jetpack-publicize-gutenberg.php}
+ *
+ * @since  5.9.1
+ */
+
+/**
+ * External dependencies
+ */
+import React, { Component } from 'react';
+import { compose } from 'redux';
+
+/**
+ * Internal dependencies
+ */
+const { __ } = wp.i18n;
+const {
+	withSelect,
+	withDispatch,
+	select,
+	dispatch,
+} = wp.data;
+import PublicizeConnection from './publicize-connection'
+
+class PublicizeForm extends Component {
+	constructor( props ) {
+		super( props );
+		let { connections } = this.props;
```

Comment:
Why is this a `let`, not a `const`?

Reply:
I'm pretty sure it was leftover from an in-progress version of that code. It clearly should be `const` now though! This is now fixed.